rules/manager.go

```diff
@@ -266,22 +282,12 @@ func typeForRule(r Rule) ruleType {
 	panic(fmt.Errorf("unknown rule type: %T", r))
 }
 
-// Eval runs a single evaluation cycle in which all rules are evaluated in parallel.
-// In the future a single group will be evaluated sequentially to properly handle
-// rule dependency.
+// Eval runs a single evaluation cycle in which all rules are evaluated sequentially.
 func (g *Group) Eval(ts time.Time) {
-	var (
-		wg sync.WaitGroup
-	)
-
 	for i, rule := range g.rules {
```

Comment:
I don't see why we need to. This is all going sequentially and the function wouldn't exit until all the rules are evaluated.